doc/src/base/numbers.md

```diff
@@ -147,10 +147,11 @@ Base.@uint128_str
 ## [BigFloats and BigInts](@id BigFloats-and-BigInts)
 
 The [`BigFloat`](@ref) and [`BigInt`](@ref) types implements
-arbitrary-precision floating point and integer arithmetic, respectively. For
-[`BigFloat`](@ref) the [GNU MPFR library](https://www.mpfr.org/) is used,
-and for [`BigInt`](@ref) the [GNU Multiple Precision Arithmetic Library (GMP)]
-(https://gmplib.org) is used.
+arbitrary-precision floating-point and integer arithmetic, respectively. For
+[`BigInt`](@ref) the [GNU Multiple Precision Arithmetic Library (GMP)]
+(https://gmplib.org) is used, and for [`BigFloat`](@ref) the
+[GNU MPFR library](https://www.mpfr.org/) is used. See also
+the [ArbNumerics.jl](https://github.com/JeffreySarnoff/ArbNumerics.jl) package.
```

Comment:
There's also Arblib.jl: https://juliahub.com/ui/Packages/General/Arblib

Reply:
Linking to another library here is already a bit dubious but at least it is a pure Julia one.

Reply:
Both packages wrap the Arb C library: https://juliahub.com/ui/Packages/General/Arb_jll

Reply:
Okay, then I don't think we need to do this at all.